synthtool/gcp/templates/java_library/.github/CODEOWNERS

```diff
@@ -6,7 +6,9 @@
 {% if 'codeowner_team' in metadata['repo'] %}
 # The {{ metadata['repo']['codeowner_team'] }} is the default owner for changes in this repo
 *                       @googleapis/yoshi-java {{ metadata['repo']['codeowner_team'] }}
+{% if 'library_type' in metadata['repo'] and metadata['repo']['library_type'] != 'GAPIC_AUTO' %}
```

Comment:
suggest adding a comment of something like 

# for handwritten libraries keep {{ metadata['repo']['codeowner_team'] }} as owner

Reply:
Done.